.doc_gen/metadata/cross_metadata.yaml

```diff
@@ -1,4 +1,16 @@
 # zexi 0.2.0
+cross_SnsPublishSubscription:
+  title: Creating an &SNSlong; Publish and Subscription Web Application that Translates Messages
+  title_abbrev: Building an &SNS; Sub Web Application 
+  synopsis: create a web application that has subscription and publish functionality 
```

Comment:
create a web application that has subscription and publish functionality and translates messages.
* Need punctuation at end of synopsis.
* Added translation feature mention because it's a key part of the example.

Reply:
fixed